execnet/gateway.py

```diff
@@ -111,22 +110,27 @@ def remote_exec(self, source, **kwargs):
             executing code.
         """
         call_name = None
+        file_name = None
         if isinstance(source, types.ModuleType):
-            linecache.updatecache(inspect.getsourcefile(source))
```

Comment:
Any reason why you dropped `linecache.updatecache`?

Reply:
Hi.

The `linecache.updatecache()` now runs in the slave process, so it saves some cpu cycles in the main process, will pin less stuff in memory, and is less stuff to pickle and pipe through to the slave. So, a small optimization.

Reply:
I see, thanks!